SpringMVCAnnotationOnlineStore/bin/TestScript.sql

```diff
@@ -0,0 +1,77 @@
+CREATE DATABASE db_test;
+USE db_test;
```

Comment:
What is this for?

Reply:
This file is now irrelevant. Only exists for initial testing. I'll exclude it going forward.